src/app/zap-templates/zcl/data-model/chip/icd-management-cluster.xml

```diff
@@ -90,12 +90,12 @@ limitations under the License.
         </command>
 
         <command source="server" code="0x01" name="RegisterClientResponse" optional="false" disableDefaultResponse="true">
-            <description> RegisterClient response command </description>
+            <description> RegisterClient response command</description>
```

Comment:
if we remove trailing blank ... how about removing the prefix blank too?

Reply:
that wouldn't hurt, done.